requirements/specifications/lifecycle/index.md

```diff
@@ -0,0 +1,675 @@
+# App Lifecycle Management
+
+Document Status: Proposed Specification
+
+See [Firebolt Requirements Governance](../../governance.md) for more info.
+
+| Contributor    | Organization   |
+| -------------- | -------------- |
+| Andrew Bennet            | Sky            |
+| Tim Dibben            | Sky            |
+| Jeremy LaCivita            | Comcast            |
+| Kevin Pearson            | Comcast            |
+| Peter Yu            | Comcast           |
+
+## 1. Overview
+
+This document describes the requirements that Firebolt platforms must
+fulfill when managing App Lifecycles. *App Lifecycle* refers to the
+lifecycle of an individual app from the time it is launched/loaded to
+the time it is unloaded and all runtime resources are discarded.
+
+*Loading* an app refers to fetching the initial resources, e.g. the
+app-manifest and index.html, and loading them into a container capable
+of running the app, e.g. a web browser.
+
+*Launching* an app refers to the process of getting an app into a state
+where it is some part of the user experience (e.g. visible, audible, or
+responding to input). This may include *loading* the app first, if
+needed.
+
+*Closing* an app refers to the process of getting an app out of a state
+where it is the primary user experience (e.g not visible, not audible,
+and not responding to input). This **does not** involve *unloading* the
+app.
+
+*Unloading* an app refers to the process of notifying an app to do any
+needed clean up, and then *terminating* it.
+
+*Terminating* an app refers to shutting down the app's container and
```

Comment:
*Backgrounding* *Suspending* *Unsuspending* are missing in this Overview section to complete the whole picture.